app/controllers/export_controller.rb

```diff
@@ -51,19 +51,19 @@ def cancel
   def progress
     map = Map.find params[:id]
     export = map.export
-    if export.present?
-      if  export.status == 'complete'
-        output = 'complete'
-      elsif export.status == 'none'
-        output = 'export not running'
-      elsif export.status == 'failed'
-        output = 'export failed'
-      else
-        output = export.status
-      end
-    else
-      output = 'export has not been run'
-    end
+    output = if export.present?
```

Comment:
Does something like this work?
![code](https://user-images.githubusercontent.com/42987854/58295766-398c0980-7da7-11e9-9b6c-49786d4cbad7.JPG)